### ISSUE TITLE
Dimensions and or Editable CAD Files 

### ISSUE THREAD

**Issue by SamSatchwell** [OpenBCI/ultracortex#29]
Hello, 
I am working on printing the Ultracortex right now but, would like to make some changes to the electrode holder. I read a previous ticket that you guys were looking into providing IGS files for some of the parts. If this is not the case, would it be in realm of possibility to get a drawing or dimension sheet of some of the parts. 

Thank you very much,

Sam

**Comment by ThreeForm** [comment]
Yes, they're certainly available.  I'm an outside contractor so I'm not able to put them on GitHub, but get in touch with contact@openbci.com and they'll be able to help.
